src/main/java/edu/greenblitz/tobyDetermined/IsRobotReady.java

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
What is this file? supposed to be some kind of "main" state machine?

Reply:
it was made to make sure that the extender was reseted

Reply:
was create to make sure that the extender is reseted

Reply:
Wait, this file is only for the extender? So I don't think the name is good,

Reply:
ok, i made a mistake. this class is a lot of function that says if the robot is ready for placing cone automaticly and if the chasiss is in the right place.
it's like isAtAngle() and isAtLength() at the same class.